prepare_vscode.sh

```diff
@@ -85,13 +84,15 @@ fi
 
 setpath() {
   { set +x; } 2>/dev/null
-  echo "$( cat "${1}.json" | jq --arg 'path' "${2}" --arg 'value' "${3}" 'setpath([$path]; $value)' )" > "${1}.json"
+  jsonTmp=$(jq --arg 'path' "${2}" --arg 'value' "${3}" 'setpath([$path]; $value)' "${1}.json")
```

Comment:
make declare `jsonTmp` as local

Reply:
For line 171, I will need to unset `jsonTmp` since this process will take place outside of a function.
```bash
jsonTmp=$( jq -s '.[0] * .[1]' product.json ../product.json )
echo "${jsonTmp}" > ../product.json
unset jsonTmp
```